src/pages/home/sidebar/SidebarView.js

```diff
@@ -15,44 +13,29 @@ const propTypes = {
     // Safe area insets required for mobile devices margins
     insets: SafeAreaInsetPropTypes.isRequired,
 
-    // Current state of the chat switcher (active of inactive)
+    // when the chat switcher is selected
     isChatSwitcherActive: PropTypes.bool,
 
-    // Current state of the CreateMenu component (active or inactive)
-    isCreateMenuActive: PropTypes.bool.isRequired,
-
-    // Callback to fire on request to toggle the CreateMenu
-    toggleCreateMenu: PropTypes.func.isRequired,
-
-    // Callback to fire when a CreateMenu item is selected
-    onCreateMenuItemSelected: PropTypes.func.isRequired,
+    // Callback to fire on avatar click
+    onAvatarClick: PropTypes.func.isRequired,
 };
 
 const defaultProps = {
     isChatSwitcherActive: false,
 };
 
 const SidebarView = props => (
-    <>
-        <View style={[styles.flex1, styles.sidebar]}>
-            <SidebarLinks
-                onLinkClick={props.onLinkClick}
-                insets={props.insets}
-            />
-            {!props.isChatSwitcherActive && (
-            <SidebarBottom insets={props.insets} />
-            )}
-            <FAB
-                isActive={props.isCreateMenuActive}
-                onPress={props.toggleCreateMenu}
-            />
-        </View>
-        <CreateMenu
-            onClose={props.toggleCreateMenu}
-            isVisible={props.isCreateMenuActive}
-            onItemSelected={props.onCreateMenuItemSelected}
```

Comment:
This is a bad merge. We are removing several things that should not be removed 🤔